src/errors/test-run/templates.js

```diff
@@ -146,6 +158,22 @@ export default {
         ${err.getCallsiteMarkup()}
     `),
 
+    [TYPE.uploadElementIsNotFileInputError]: err => markup(err, `
+        The upload action selector does not match a file input element.
```

Comment:
Make texts action-agnostic, so we can potentially reuse errors. So it wil be:
`Action selector does not match a file input element.`